packaging/flavorgen/flavors/generators.go

```diff
@@ -398,6 +398,18 @@ func kubeVIPPod() string {
 							Name:  "vip_interface",
 							Value: "eth0",
 						},
+						{
```

Comment:
> sets the leader election params to the default values

I see different defaults [here](https://github.com/plunder-app/kube-vip/pull/73/files#diff-280a01902e7f52d4ee3966850378b8a7R42-R44). were the defaults in this PR meant to be different from the ones in kube-vip?

Reply:
Those above are the defaults for `kube-vip` the defaults for k\k are longer, which are what are being used here.